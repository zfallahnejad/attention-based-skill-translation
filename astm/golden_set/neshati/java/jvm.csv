139985,jvm
3448419,jvm
22656,jvm
1103872,jvm
40342,jvm
605744,jvm
12960,jvm
571407,jvm
