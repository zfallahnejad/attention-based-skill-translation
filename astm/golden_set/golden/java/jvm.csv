3448419,45,35,0.5625
22656,40,34,0.5405405405405406
1103872,21,20,0.5121951219512195
40342,18,15,0.5454545454545454
605744,15,6,0.7142857142857143
12960,13,15,0.4642857142857143
571407,12,12,0.5
157247,10,4,0.7142857142857143
